bitnami/ghost/templates/deployment.yaml

```
@@ -154,10 +154,12 @@ spec:
               value: {{ .Values.ghostBlogTitle | quote }}
             - name: GHOST_ENABLE_HTTPS
               value: {{ ternary "yes" "no" .Values.ghostEnableHttps | quote }}
-            - name: GHOST_EXTERNAL_HTTP_PORT_NUMBER
-              value: {{ coalesce .Values.service.ports.http .Values.service.port| quote }}
-            - name: GHOST_EXTERNAL_HTTPS_PORT_NUMBER
-              value: {{ coalesce .Values.service.ports.https .Values.service.httpsPort| quote }}
+            {{- if eq ( toString ( .Values.service.type )) "LoadBalancer" }}
```

Inner parenthesis are not needed
```suggestion
            {{- if eq (toString .Values.service.type) "LoadBalancer" }}
```